lizmap/modules/lizmap/lib/Request/OGCRequest.php

```diff
@@ -121,9 +121,16 @@ public function parameters()
             ));
         }
 
-        // Provide user and groups to lizmap plugin access control
+        // Provide user and groups to lizmap plugin access control without private group
         $user = $appContext->getUserSession();
         $userGroups = $appContext->aclUserGroupsId();
+        $userPrivateGroup = $appContext->aclUserPrivateGroup();
+        if ($userPrivateGroup) {
```

Comment:
there is no need to do this if statement, if we call the new proposed method `aclPublicGroupsIdByUser()` instead of `aclUserGroupsId()`.